ledger-core/v2/network/facade/facade.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/insolar/blob/master/LICENSE.md.
+
+package facade
+
+import (
+	"context"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/bus"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/v2/pulse"
+
+	"github.com/pkg/errors"
+)
+
+type Marshaler interface {
+	Marshal() ([]byte, error)
+	// Temporary commented to look like insolar.Payload
+	//MarshalHead() ([]byte, error)
+	//MarshalContent() ([]byte, error)
+}
+
+type option struct {
+	syncBody bool
+}
+
+type SendOption func(*option)
+
+func WithSyncBody() SendOption {
+	return func(o *option) {
+		o.syncBody = true
+	}
+}
+
+//go:generate minimock -i github.com/insolar/assured-ledger/ledger-core/v2/network/facade.Messenger -o ./ -s _mock.go -g
+
+type Messenger interface {
```

Comment:
Interfaces should be defined on /network/ layer, as far as I think. @AndreyBronin whats your opinion?